msal/src/main/java/com/microsoft/identity/client/controllers/MSALAcquireTokenOperationParameters.java

```diff
@@ -5,12 +5,14 @@
 
 import com.microsoft.identity.client.UiBehavior;
 import com.microsoft.identity.client.User;
+import com.microsoft.identity.common.internal.cache.MsalOAuth2TokenCache;
 
 import java.util.List;
 
-public class MSALAcquireTokenRequest {
+public class MSALAcquireTokenOperationParameters extends MSALOperationParameters {
 
     private Context mAppContext;
+    private MsalOAuth2TokenCache mTokenCache;
     private Activity mActivity;
     private List<String> mScopes;
     private User mUser;
```

Comment:
Do you have everything you need on [`com.microsoft.identity.client.IAccount`](https://github.com/AzureAD/microsoft-authentication-library-for-android/blob/dev/msal/src/main/java/com/microsoft/identity/client/IAccount.java) such that this object can be swapped-out in the near-future?

Reply:
Why would I swap this out?

Reply:
Ah you mean User.... yes... currently this isn't being used... since i'm working on AcquireToken.... but yes.  I'll be ready to swap out.